apps/backend/app/services/contact_files.py

```
@@ -0,0 +1,99 @@
+    Mongo copy backs this up for uploads, so a Redis outage must not fail
+    the request."""
+    try:
+        redis = from_url(settings.redis_url, decode_responses=True)
+        await redis.ping()  # Minimal connection check
+        await redis.set(
+            cache_key(file_ref, owner_id),
+            json.dumps([r.model_dump() for r in rows]),
+            ex=CACHE_TTL_SECONDS,
+        )
+        await redis.aclose()
+    except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)